csrankings-r.csv

```diff
@@ -3,6 +3,7 @@ R. A. Bodnar,University of Southampton,https://www.ecs.soton.ac.uk/people/rab1e1
 R. B. Mishra,IIT (BHU) Varanasi,http://www.iitbhu.ac.in/cse/index.php/people/faculty/24.html,5ujlOtgAAAAJ
 R. Benjamin Shapiro,University of Colorado Boulder,http://benshapi.ro,BjX6lbwAAAAJ
 R. Chandrasekaran,University of Texas at Dallas,https://www.utdallas.edu/~chandra,NOSCHOLARPAGE
+R. Chunara,New York University,http://rumichunara.github.io,7NhhkR8AAAAJ

```

Comment:
DBLP name is "Rumi Chunara", please fix. (https://dblp.org/pid/130/0470.html)

Reply:
Fixed dblp name